tracer/src/Datadog.Trace/Debugger/Symbols/SymbolsExtensions.cs

```diff
@@ -69,4 +70,14 @@ internal static bool IsInterfaceType(this TypeDefinition typeDefinition)
         var typeAttributes = typeDefinition.Attributes;
         return (typeAttributes & TypeAttributes.Interface) == TypeAttributes.Interface;
     }
+
+    internal static bool IsHiddenThis(this Parameter parameter)
+    {
+        return parameter.SequenceNumber == -2;
```

Comment:
Comment to explain this magic number?

Reply:
We will continue to work on this in a later PR